helios-tools/src/test/java/com/spotify/helios/cli/CliConfigTest.java

```diff
@@ -33,6 +34,14 @@
 
   @Rule public final ExpectedException expectedEx = ExpectedException.none();
 
+  @Before
```

Comment:
If you want to make sure `environment` is always empty unless explicitly set, should you not do this in `@After` also?